data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/KafkaSourceSecurityConfigurer.java

```diff
@@ -234,11 +245,14 @@ public static String getBootStrapServersForMsk(final AwsIamAuthConfig awsIamAuth
         }
     }
 
-    public static void setAuthProperties(Properties properties, final KafkaSourceConfig sourceConfig, final Logger LOG) {
+    public static GlueSchemaRegistryKafkaDeserializer setAuthProperties(Properties properties, final KafkaSourceConfig sourceConfig, final Logger LOG) {
```

Comment:
This method approach is a little problematic. It is doing two things: 1) setting authorization properties; and 2) creating the deserializer and returning it.

This code you created seems to be something that could go into another method entirely.

Reply:
Yes. Hardeep mentioned the same. I will do it in the next PR